tests/scripts/single_card/run_demos_single_card_wh_common.sh

```diff
@@ -12,9 +12,5 @@ fi
 WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest -n auto --disable-warnings -q -s --input-method=cli --cli-input="YOUR PROMPT GOES HERE!"  models/demos/wormhole/falcon7b/demo_wormhole.py::test_demo[wormhole_b0-True-user_input0-1-default_mode_1024_stochastic]
 # llama3.1-8B
 WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest -n auto models/demos/wormhole/llama31_8b/demo/demo.py --timeout 600
-WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest -n auto models/demos/wormhole/llama31_8b/demo/demo_with_prefill.py --timeout 600
```

Comment:
This means we no longer some of these models on N150.

For example, it seems the only mamba test we have is now no longer on N150.

What's the policy on this, and do we have buy in from customer team? @uaydonat @dvartaniansTT @mbahnasTT

Reply:
@tt-rkim I added back the n150 tests while keeping the benchmark uploads only for n300 perf